DLaB.AttributeManager/AttributeManagerPlugin.cs

```diff
@@ -1043,6 +1146,35 @@ private void SelectAllKeyDownHandler(object sender, KeyEventArgs e)
                 e.Handled = true;
             }
         }
+
+        private void grpSteps_Enter(object sender, EventArgs e)
+        {
+
+        }
+
+        private int? getPrecisionSetting()
+        {
+            int tmp;
+            int? number = null;
+            if (int.TryParse(numAttPrecisionTxt.Text, out tmp))
+            {
+                number = tmp;
+            }
+            return number;
+        }
+
+        private void chkIgnoreUpdateErrors_CheckedChanged(object sender, EventArgs e)
+        {
+            if (chkIgnoreUpdateErrors.Checked)
+            {
+                //warn about possible data loss
+                DialogResult result = MessageBox.Show("Selecting this option may result in significant data loss if there are read-only records or invalid data", "Warning", MessageBoxButtons.OKCancel, MessageBoxIcon.Warning);
```

Comment:
This just means that some of the data will not get migrated, because of the error, and just continue correct?